benchmark.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Comment:
**issue** needed to run `chmod a+x benchmark.py` so that I could have executed the script by `./benchmark.py`. The script should be commited with executable flag.